in.cinny.Cinny.yml

```diff
@@ -0,0 +1,37 @@
+id: in.cinny.Cinny
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: cinny
+rename-icon: cinny
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  - name: Cinny
+    buildsystem: simple
+    build-commands:
+      - ar x cinny_*.deb
+      - rm -f cinny_*.deb
+      - tar xf data.tar.gz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r usr/bin/cinny /app/bin
+      - mkdir -p /app/share/icons/hicolor
+      - cp -r usr/share/icons/hicolor/* /app/share/icons/hicolor
+      - rm -rf /app/share/icons/hicolor/512x512@2x
+      - install -Dm744 usr/share/applications/cinny.desktop /app/share/applications/in.cinny.Cinny.desktop
+    sources:
+      - url: https://github.com/cinnyapp/cinny-desktop/releases/download/v2.0.2/cinny_2.0.2_amd64.deb
```

Comment:
Please configure https://github.com/flathub/flatpak-external-data-checker/ to automate updates.

Reply:
Application has auto-updates in itself, do I still need to configure this?

Reply:
you probaly need to disable them. The content of the flatpak IS read-only.

Reply:
so if I get it correctly, that's only possible if I build flatpak from the source? (or it can be done for .deb too?)